Dockerfile

```diff
@@ -1,20 +1,20 @@
-FROM golang:1.18-alpine3.14  AS build-env
+FROM golang:1.18  AS build-env
 
 RUN echo $GOPATH
-
-RUN apk add --no-cache git gcc musl-dev
-RUN apk add --update make
+RUN apt update
+RUN apt install git gcc musl-dev make -y
 RUN go install github.com/google/wire/cmd/wire@latest
 WORKDIR /go/src/github.com/devtron-labs/devtron
 ADD . /go/src/github.com/devtron-labs/devtron/
 RUN GOOS=linux make build-all
 
 # uncomment this post build arg
-FROM alpine:3.15.0 as  devtron-all
-RUN apk add --no-cache ca-certificates
-RUN apk update
-RUN apk add git
-RUN apk add curl
+FROM ubuntu as  devtron-ea
+
+RUN apt update
+RUN apt install ca-certificates git curl -y
+RUN apt clean autoclean
```

Comment:
should we use `autoremove` in all dockerfile?

Reply:
already there